src/components/categories/Create.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React, { useContext } from 'react';
+// import { useNavigate } from 'react-router-dom';
+import FormContext from '../../Context/FormContext';
+import { useAddCategoryMutation } from '../../Redux/APIFunctions/categories';
+import Form from './Form';
+
+const CreateCategory = () => {
+  const { values, setShowForm } = useContext(FormContext);
+  const [addCategory] = useAddCategoryMutation();
+  // const navigate = useNavigate();
```

Comment:
- [ ] Please, kindly remove this old piece of commented code, to improve the cleanliness of the repo 💪